ui/src/components/RolesTable/index.jsx

```diff
@@ -30,7 +30,7 @@ export default class RolesTable extends Component {
       pageInfo,
     }).isRequired,
     onPageChange: func.isRequired,
-    /** A search term to refine the list of roles */
+    /** Value of search field */
```

Comment:
Was this changed by mistake?

Reply:
Also, let's add a comment if possible for all places we are adding `searchTerm` in propTypes.